object_test.go

```diff
@@ -659,3 +659,125 @@ func TestIssue150_EmbedStructJavaDecode(t *testing.T) {
+
+	decoder := NewDecoderWithSkip(data[16:])
+
+	var field interface{}
+	var err error
+	var ok bool
+	var str string
+	var attachments map[string]string
+	attr := &dubboAttr{}
+
+	field, err = decoder.Decode()
+	if err != nil {
+		t.Errorf("Failed to decode dubbo framework version, err=%v\n", err)
+	}
+
+	if field != nil {
+		str, ok = field.(string)
+		if !ok {
+			t.Errorf("Failed to decode dubbo framework version, illegal type\n")
+		}
+	}
+	attr.dubboVersion = str
+
+	field, err = decoder.Decode()
+	if err != nil {
+		t.Errorf("Failed to decode dubbo path, err=%v\n", err)
+	}
+
+	// check path is nil ?
+	if str = ""; field != nil {
```

Comment:
hey, guy. to make the codes more readable, this line may should be `str = "" \n if field != nil`. and the following codes should be changed too.